### ISSUE TITLE
Function call

### ISSUE THREAD

**Issue by atis-s** [samuelgjekic/OpenAI-Assistants-API-Wrapper-for-.NET#9]
Function calling seems not to work.
There need to be true / false response to the API.

**Comment by samuelgjekic** [comment]
> Function calling seems not to work.
> There need to be true / false response to the API.

Yes i have not had the time to look into this, i could maybe check it out tonight, however if you feel like this is something you could help with, feel free to do it 🙂 

Link to the official docs:
https://platform.openai.com/docs/assistants/tools/function-calling?lang=curl

Edit: It also seems that we would have to define the functions when we create the assistant so this would have to be implemented in the method this wrapper uses to create assistants.
